src/pim-types.ts

```diff
@@ -255,6 +264,57 @@ export class TaskType extends EventType {
     ret.color = this.color;
     return ret;
   }
+
+  public getNextOccurence(): TaskType | null {
+    if (!this.isRecurring()) {
+      return null;
+    }
+
+    const rrule = this.rrule.clone();
+
+    if (rrule.count && rrule.count <= 1) {
```

Comment:
Also, I see here you are using the `rrule.count != null` rather than `isByCount`... See my other comment.